client/js/widgets/widget.js

```diff
@@ -2302,6 +2312,18 @@ export class Widget extends StateManaged {
       await this.set('rotation', degrees);
   }
 
+  async selected() {
+    if (!this.addOverlay || window.getComputedStyle(addOverlay).display === 'none')
```

Comment:
`addOverlay` should be undefined. `this.addOverlay` might work.

`window.getComputedStyle` should be unnecessary. Checking `this.addOverlay.style.display` should be enough.